WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt

```diff
@@ -162,8 +162,11 @@ class SitePreviewViewModel @Inject constructor(
     private fun startCreateSiteService(previousState: SiteCreationServiceState? = null) {
         if (networkUtils.isNetworkAvailable()) {
             siteCreationState.apply {
+                // A non-null [segmentId] may invalidate the [siteDesign] selection
+                // https://github.com/wordpress-mobile/WordPress-Android/issues/13749
+                val segmentIdentifier = if (siteDesign != null) null else segmentId
                 val serviceData = SiteCreationServiceData(
-                        segmentId,
+                        segmentIdentifier,
```

Comment:
_For reference if another reviewer was wondering: this change comes from https://github.com/wordpress-mobile/WordPress-Android/pull/13751 (i.e. even if there was no new beta yet after this, this change is still expected)_